config/vufind/config.ini

```diff
@@ -528,6 +528,8 @@ disable_from = false
 ; From field override. Setting this allows keeping the "from" field in email forms
 ; but will only use it as a reply-to address. The address defined here is used as the
 ; actual "from" address.
+; Note: If a feature explicitly sets a different reply-to address (for example,
+; Feedback forms), the original from address will NOT override that reply-to value.
```

Comment:
I reworded this slightly in an effort to make it more clear. Please let me know if this is okay with you.

Reply:
Looks good to me.